### PR TITLE
Completed html-forms exercise. - Barbara Liao

### PR DESCRIPTION
**Pull request by barbara-liao**
html-forms screenshot

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        </label>
+        <label for="CSS">
+          <input type="radio" name="language" value="CSS" id="CSS" /> CSS
+        </label>
+        <label for="JavaScript">
+          <input type="radio" name="language" value="JavaScript" id="JavaScript" /> JavaScript
+        </label>
+      </div>
+      <div>
+        <label for="sign-up">Would you like to sign up for more information?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:37]: This can actually be just a regular text element, such as a `p`.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <div><br /></div>
+        <input type="checkbox" name="information" value="sign-up" id="sign-up" /> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:39]: _This_ "Sign me up!" text should be the `label` here.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! A few notes for you:

**Comment by barbara-liao** [comment]
Thanks for the notes! I pushed the updated file :)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**